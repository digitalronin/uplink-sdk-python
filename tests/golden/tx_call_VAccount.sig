r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 98232085871255060402046161019820634587702997670875301346428314644739558332236
