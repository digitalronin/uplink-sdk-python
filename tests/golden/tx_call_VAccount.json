{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VAccount", 
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE05ODIzMjA4NTg3MTI1NTA2MDQwMjA0NjE2MTAxOTgyMDYzNDU4NzcwMjk5NzY3MDg3NTMwMTM0NjQyODMxNDY0NDczOTU1ODMzMjIzNg==", 
    "timestamp": 1231006505, 
    "to": null
}